src/MonC/Codegen/ILListingWriter.cs

```diff
@@ -1,18 +1,17 @@
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using MonC.Bytecode;
-using MonC.Codegen;
 
-namespace MonC.Frontend
+namespace MonC.Codegen
 {
-    public class IntermediateLanguageWriter
+    public class ILListingWriter
```

Comment:
This class name really demonstrates the shortcomings of the defacto C# interface naming scheme :P